README.md

```diff
@@ -39,7 +34,7 @@ Charge stations can use either OCPP 1.6j or OCPP 2.0.1.
 For TLS, the charge station should use a certificate provisioned using the
 [Hubject CPO EST service](https://hubject.stoplight.io/docs/open-plugncharge/486f0b8b3ded4-simple-enroll-iso-15118-2-and-iso-15118-20).
 
-A charge station must first be registered with the CSMS before it can be used. This can be done using the
+A charge station must register with Maeve before use, via the
```

Comment:
The charge station doesn't register. It needs to be registered by someone else.

Reply:
Not entirely sure what you mean

Reply:
The registration is not something the CS software can do. That's what now seems to be implied by removing the passive voice.

Reply:
Could you suggest better text please?